File: xd.py
def final():
    print("Agarro mi mochila")
    print("Me subo al carro")
    print("A chambear")

print("Me levanto")
respuesta_banho = input("¿Me bañé?: 'Y/N' ")

if respuesta_banho == "Y":
    print("Me cambio")
    respuesta_media_hora = input("¿Tengo más de media hora?: 'Y/N' ")
    if respuesta_media_hora == "Y":
        print("Desayuno")
    final()
else:
    print("Me baño")
    print("Me cambio")
    respuesta_media_hora = input("¿Tengo más de media hora?: 'Y/N' ")
    if respuesta_media_hora == "Y":
        print("Desayuno")
    final()

File: xd.js
const readline = require("readline");
const readl = readline.createInterface({ input: process.stdin, output: process.stdout, escapeCodeTimeout: 200})
function final() {console.log("Agarro mi mochila"), console.log("Me subo al carro")}
console.log("Me levanto")
readl.question("¿Me bañe?: ´Y/N´", (respuesta) => { if(respuesta == "Y"){console.log("Me cambio"), readl.question("¿Tengo mas de medio hora?: ´Y/N´", (respuesta) => { if (respuesta == "Y"){console.log("desayuno")} final()})} else {console.log("Me baño"), console.log("Me cambio"), readl.question("¿Tengo mas de medio hora?: ´Y/N´", (respuesta) => { if (respuesta == "Y"){console.log("desayuno")} final()})}})